igc_lib.py

```diff
@@ -57,50 +52,67 @@ def rawtime_float_to_hms(timef):
     
     return hms((time/3600), (time%3600)/60, time%60)
 
-def degrees_float_to_degrees_minutes_seconds(dd):
-    """Converts time from floating point degrees to degrees/minutes/floating point seconds.
+def degrees_float_to_degrees_minutes_seconds(dd, lon_or_lat =''):
```

Comment:
Do we really need the case where lon_or_lat is left unspecified? Maybe let's make it a required argument, then you'd be able to remove some of the code below.